src/helpers/slack.ts

```diff
@@ -10,29 +11,59 @@ import {
 const fs = require("fs");
```

Comment:
[Non blocking comment]

Since TS is being used, its good to follow convention at hand

`import fs from 'fs';`

Reply:
Updated